CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettings.java

```diff
@@ -0,0 +1,502 @@
+        final Color selectedLowerDarkBlue = new Color(80, 160, 255);
+        final Color unselectedUpperGray = new Color(225, 225, 225);
+        final Color unselectedLowerGreyBlue = new Color(175, 185, 195);
+
+        try {
+            // To apply new tab colors to a Windows LAF
+            // we need to find the current TabDisplayerUI
+            // and force it to re-read from a revised set of colors
+            Class<?> displayerClass = (Class<?>) UIManager.getDefaults()
+                    .get("org.netbeans.swing.tabcontrol.plaf.Windows8VectorViewTabDisplayerUI");
+            if (displayerClass != null) {
+                final Class<?> tabDisplayer = displayerClass.getSuperclass();
+
+                if (tabDisplayer == null) {
+                    LOGGER.info(" >>> Unable to apply Tab coloring to Windows LAF : no superclass for displayerClass :");
+                    ouputUIDefaultValues(null, null);
+                    return;
+                }
+                
+                // get Reflection object before updating any UI settings
+                final Field colsReady = tabDisplayer.getDeclaredField("colorsReady"); //NOSONAR
+
+                // Update the UIManager with settings appropriate for Windows (8+) LAF
+                UIManager.getDefaults().put("tab_sel_fill", activeMidSectionBlue);
+                UIManager.getDefaults().put("tab_focus_fill_upper", selectedUpperLightBlue);
+                UIManager.getDefaults().put("tab_focus_fill_lower", selectedLowerDarkBlue);
+                UIManager.getDefaults().put("tab_unsel_fill_upper", unselectedUpperGray);
+                UIManager.getDefaults().put("tab_unsel_fill_lower", unselectedLowerGreyBlue);
+                UIManager.getDefaults().put("tab_mouse_over_fill_upper", selectedUpperLightBlue);
+                UIManager.getDefaults().put("tab_mouse_over_fill_lower", activeMidSectionBlue);
+                UIManager.getDefaults().put("tab_attention_fill_upper", activeMidSectionBlue);
+                UIManager.getDefaults().put("tab_attention_fill_lower", selectedLowerDarkBlue);
+                
+                // reset static field "colorsReady" value to false
+                colsReady.setAccessible(true); //NOSONAR
+                colsReady.setBoolean(tabDisplayer, false); //NOSONAR
+            } else {
+                // check if XP settings are being used
+                displayerClass = (Class<?>) UIManager.getDefaults()
+                    .get("org.netbeans.swing.tabcontrol.plaf.WinXPViewTabDisplayerUI");
+                if (displayerClass != null) {
+                    // get Reflection objects before updating any UI settings
+                    final Field colsReady = displayerClass.getDeclaredField("colorsReady"); //NOSONAR
+                    Class<?>[] argList = null;
+                    final Method initCols = displayerClass.getDeclaredMethod("initColors", argList);
```

Comment:
if this is the only use of `argList`, could you potentially just pass null directly into this `getDeclaredMethod` call instead? You could then remove `argList`

Reply:
Done